flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/SourceStreamTask.java

```diff
@@ -89,7 +89,8 @@ public void triggerCheckpoint(long checkpointId) throws FlinkException {
 					// TODO - we need to see how to derive those. We should probably not encode this in the
 					// TODO -   source's trigger message, but do a handshake in this task between the trigger
 					// TODO -   message from the master, and the source's trigger notification
-					final CheckpointOptions checkpointOptions = CheckpointOptions.forCheckpointWithDefaultLocation();
+					final CheckpointOptions checkpointOptions = CheckpointOptions.
```

Comment:
nit: formatting  only make the argument `getConfiguration().isExactlyOnceCheckpointMode()` as as separate line